docs/content/components/dropdown.md

```diff
@@ -1,85 +1,164 @@
 ---
 title: Dropdown
 status: Alpha
-source: https://github.com/primer/view_components/tree/main/app/components/primer/dropdown_component.rb
+source: https://github.com/primer/view_components/tree/main/app/components/primer/dropdown.rb
 storybook: https://primer.style/view-components/stories/?path=/story/primer-dropdown-component
 ---
 
 import Example from '../../src/@primer/gatsby-theme-doctocat/components/example'
+import RequiresJSFlash from '../../src/@primer/gatsby-theme-doctocat/components/requires-js-flash'
+
+<RequiresJSFlash />
 
 <!-- Warning: AUTO-GENERATED file, do not edit. Add code comments to your Ruby instead <3 -->
 
 `Dropdown` is a lightweight context menu for housing navigation and actions.
-They're great for instances where you don't need the full power (and code) of the select menu.
+They're great for instances where you don't need the full power (and code) of the SelectMenu.
 
 ## Examples
 
 ### Default
 
-<Example src="<div>  <details data-view-component='true' class='dropdown details-overlay details-reset position-relative'>  <summary role='button' data-view-component='true' class='btn'>              Dropdown  </summary>  <div data-view-component='true'>    <details-menu role='menu' data-view-component='true' class='dropdown-menu dropdown-menu-se'>    <div class='dropdown-header'>      Options    </div>  <ul>      <li data-view-component='true' class='dropdown-item'>Item 1</li>      <li data-view-component='true' class='dropdown-item'>Item 2</li>      <li role='none' data-view-component='true' class='dropdown-divider'></li>      <li data-view-component='true' class='dropdown-item'>Item 3</li>      <li data-view-component='true' class='dropdown-item'>Item 4</li>  </ul></details-menu></div></details></div>" />
+<Example src="<details data-view-component='true' class='dropdown details-overlay details-reset'>  <summary role='button' data-view-component='true' class='btn'>            Dropdown      </summary>  <div data-view-component='true'>    <details-menu role='menu' data-view-component='true' class='dropdown-menu dropdown-menu-se'>    <div class='dropdown-header'>      Options    </div>      <a role='menuitem' data-view-component='true' class='dropdown-item'>Item 1</a>      <a role='menuitem' data-view-component='true' class='dropdown-item'>Item 2</a>      <a role='separator' data-view-component='true' class='dropdown-divider'></a>      <a role='menuitem' data-view-component='true' class='dropdown-item'>Item 3</a>      <a role='menuitem' data-view-component='true' class='dropdown-item'>Item 4</a></details-menu></div></details>" />
```

Comment:
It looks like there's an extra div being wrapping the `details-menu`. 
Previously I think we tested and saw that the extra div was causing issues with the js selection.
Might want to verify this!

Reply:
Do you know what was the issue? docs/storybook/previews are working fine here. To remove that `div` we may need to do bigger changes to `DetailsComponent`

Reply:
When I add an extra div to the [details-menu-repo examples](https://github.github.io/details-menu-element/examples/), it looks like the summary text isn't changing as expect based on the selection. I'm not sure how this will impact dropdown though.

Reply:
I think that this is related to the `data-menu-button` prop, which we don't use in dropdowns
Given that we currently ship this component with the wrapping `div`, I'm inclined to not change that behavior in this PR and try to fix it in a follow-up. would that be ok with you?